config/variants/multi-gw/base/manager_multi_gateway_patch.yaml

```diff
@@ -13,5 +13,7 @@ spec:
         env:
         - name: CONTROLLER_KONG_ADMIN_SVC
           value: kong/kong-admin
+        - name: CONTROLLER_GATEWAY_DISCOVERY_DNS_STRATEGY
+          value: ip
```

Comment:
Do we want this explicitly in the manifest or do we prefer to rely on default defined in code? (and documented in flag's description)?

Reply:
Fair enough. Removed!